src/main/java/com/linecorp/armeria/server/docs/EnumInfo.java

```diff
@@ -22,14 +22,15 @@
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
+import java.util.Map;
 import java.util.Objects;
 
 import org.apache.thrift.meta_data.EnumMetaData;
 import org.apache.thrift.protocol.TType;
 
 class EnumInfo extends TypeInfo implements ClassInfo {
 
-    static EnumInfo of(EnumMetaData enumMetaData) {
+    static EnumInfo of(EnumMetaData enumMetaData, Map<String, String> docStrings) {
```

Comment:
Global comment: could we add a shortcut method that allows the caller to create without `docStrings`? e.g.

```java
EnumInfo of(EnumMetaData enumMetaData) {
    return of(enumMetaData, Collections.emptyMap());
}
```

.. so that we don't need to add a bunch of `Collections.emptyMap()`s in our unit tests.